docs/docsite/rst/getting_started_ee/setup_environment.rst

```diff
@@ -0,0 +1,37 @@
+.. _setting_up_environment:
+
+###########################
+Setting up your environment
+###########################
+
+Complete the following steps to set up a local environment for your first execution environment:
+
+1. Ensure the following packages are installed on your system:
+
+* ``podman`` or ``docker``
+* ``python3``
+
+If you use the DNF package manager, install them as follows:
+
+.. code-block:: bash
+
+  sudo dnf install -y podman python3
+
+2. Upgrade the `pip` package manager and install `ansible-navigator` to run EEs.
+
+Installing `ansible-navigator` lets you run EEs.
+It includes the `ansible-builder` package to build EEs.
+
```

Comment:
While it seems quicker to install navigator and get both, is it worth saying you can install builder on its own with pip install ansible-builder... or whatever the command is? @oraNod what do you thing? It complicates the getting started to mention this, but without it, someone looking for a fast/quick way to install just builder doesn't see how here.

Reply:
changed the wording, PTAL